org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -601,6 +610,28 @@ SsoApplicationManager:
         ]
       }
 
+# Role for a user that can access BedRock in the Synapse LLM prod account
+SSoLLmDeveloper:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.7/templates/SSO/aws-sso.njk
+  StackName: !Sub '${resourcePrefix}-${appName}-llmdeveloper'
+  StackDescription: 'Role used by LLM Developer group within Synapse LLM prod account'
+  TerminationProtection: false
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: false
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref SynapseLlmProdAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref SynapseLLmProdDeveloperGroup
+    permissionSetName: 'llm-developer'
+    managedPolicies:
+      - 'arn:aws:iam::aws:policy/AmazonBedrockFullAccess'
+      - 'arn:aws:iam::aws:policy/AWSCloudFormationFullAccess'
```

Comment:
why do developers need cloudformation access?

Reply:
They setup/debug using the UI then eventually move to Cloudformation for production resources.